Completion/Unix/Command/_zfs

```diff
@@ -145,7 +145,7 @@ _zfs() {
 	# set the sorting for *size properties to false by default?
 	rw_properties=(
 		"aclinherit:value:(discard noallow restricted passthrough passthrough-x)"
-		"aclmode:value:(discard mask passthrough)"
+		$(if [[ "$OSTYPE" == "linux-gnu" ]]; then echo "acltype:value:(off noacl posixacl)"; else echo "aclmode:value:(discard mask passthrough)"; fi );
```

Comment:
Thanks for the contribution. This is a useful improvement.

Using echo inside `$( ... )` creates a subshell and pipe. How about appending to rw_properties after the end of the assignment - something like:

    if [[ $OSTYPE = linux-gnu ]]; then
      rw_properties+=( "acltype:value:(off noacl posixacl)" )
    else
      rw_properties+=( ... )
    fi

Actually, while changing this, from a quick check it looks like the possible values are discard|groupmask|passthrough|restricted for openzfs (FreeBSD and Illumos) while the original form of the completion is right for Solaris. So it could be good to handle that difference.